pkg/cmd/pulumi/new.go

```diff
@@ -31,6 +31,7 @@ import (
 	surveycore "github.com/AlecAivazis/survey/v2/core"
 	"github.com/opentracing/opentracing-go"
 	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
This package is marked as deprecated and points to [golang.org/x/term](https://pkg.go.dev/golang.org/x/term).
On a first glance, it appears to be a drop-in replacement for the APIs we're using.

```suggestion
	"golang.org/x/term"
```